cmd/swctl/app/cmd_dependency.go

```diff
@@ -218,35 +274,36 @@ func linkSetUpDown(cli Cli) *cobra.Command {
 	return cmd
 }
 
-func (*Dependencies) IsDockerAvailable(cli Cli) bool {
+func IsDockerAvailable(cli Cli) (bool, error) {
 	out, _, err := cli.Exec("whereis docker", nil)
 	if err != nil {
-		panic(err)
+		return false, err
 	}
 	if strings.Contains(out, "/docker") {
-		return true
+		return true, nil
 	}
-	return false
+	return false, nil
 }
 
-func (*Dependencies) IsHugePagesEnabled(cli Cli) (bool, int) {
+func AllocatedHugePages(cli Cli) (int, error) {
 	out, _, err := cli.Exec("sysctl vm.nr_hugepages -n", nil)
 	if err != nil {
-		panic(err)
+		return 0, err
 	}
 	hugePgSize, err := strconv.Atoi(strings.TrimSpace(out))
 	if err != nil {
-		fmt.Println(err)
+		return 0, err
 	}
 	if hugePgSize == 0 {
-		return false, hugePgSize
+		return 0, err
 	}
 
-	return true, hugePgSize
+	return hugePgSize, err
```

Comment:
The same as `return hugepgSize, nil`

Reply:
Current state:
```
	if hugePgSize == 0 {
		return 0, err
	}

	return hugePgSize, nil
```
Err is already checked for not being nil, so that code is the same as
```
return hugePgSize, nil
```
I just want to make it simple so that it can be read easily and not complicate it.